graphql-parser/src/query/format.rs

```diff
@@ -249,17 +250,17 @@ impl<'a> Displayable for InlineFragment<'a>
     }
 }
 
-impl<'a> Displayable for TypeCondition<'a> 
-{
-    fn display(&self, f: &mut Formatter) {
-        match *self {
-            TypeCondition::On(ref name) => {
-                f.write("on ");
-                f.write(name.as_ref());
-            }
-        }
-    }
-}
+// impl<'a> Displayable for TypeCondition<'a> 
+// {
+//     fn display(&self, f: &mut Formatter) {
+//         match *self {
+//             TypeCondition::On(ref name) => {
+//                 f.write("on ");
+//                 f.write(name.as_ref());
+//             }
+//         }
+//     }
+// }
```

Comment:
Do you want to delete these bits all together?

Reply:
Yes, ty!